ViteMaDose/Views/CentresList/Cells/CentresTitleCell.swift

```diff
@@ -36,15 +31,15 @@ extension CentresTitleCell {
 
         guard dosesCount > 0 else {
             let title = NSMutableAttributedString(
-                string: "\(Constant.titleNoDoseText) \(countyName)",
+                string: "\(LocalizedString.VaccinationLocationsList.noResults) \(countyName)",
                 attributes: attributes
             )
             title.setColorForText(textForAttribute: countyName, withColor: .mandy)
             return title
         }
 
-        let dosesCountString = "\(String(dosesCount)) \(Constant.titleSecondPartText)"
-        let titleString = "\(Constant.titleFirstPartText) \(dosesCountString) \(Constant.titleThirdText) \(countyName)"
+        let dosesCountString = dosesCount > 1 ? LocalizedString.VaccinationLocationsList.doses(dosesCount) : LocalizedString.VaccinationLocationsList.dose(dosesCount)
```

Comment:
Same thing for formatted string here

Reply:
Alternative way: I could create a new function and make the dose/doses localized strings private, and the single function will decide accordingly what to return. What do you think about that?